python-sdk/src/astro/databases/snowflake.py

```diff
@@ -52,7 +52,6 @@
 }
 
 COPY_OPTIONS = {
-    FileType.CSV: "ON_ERROR=CONTINUE",
```

Comment:
@sunank200, could you confirm if our benchmark tests still run without this?

Reply:
I will conform but it would require this parameter to be passed as `copy_options`.

Reply:
I wonder why our synthetic dataset would require us to set this option - ideally, it would have the correct number of rows, columns and values. But I'm also happy for this to be added as a follow-up ticket!

Reply:
I think in synthetic datasets, csv had the first rows as headers and we did not have the option to skip the header implemented then. Hence, ON_ERROR=CONTINUE was there as default. 

Created a separate issue for this to do the benchmarking: https://github.com/astronomer/astro-sdk/issues/1699